config/vufind/KohaRest.ini

```diff
@@ -184,3 +184,15 @@ extraFields = item-issue:acceptTerms:pickUpLocation
 [ItemTypeRenewalBlockMappings]
 ; ILL[too_many] = "ILSMessages::ill_item_renewal_limit"
 ; ILL[too_soon] = "ILSMessages::ill_renewal_too_soon"
+
+; This section allows libraries to define different custom itemLimit rules for 
+; different biblio-level item types.
+; In Koha the biblio-level item type is defined in the 942$c subfield.
+; The string inside the [] is the Koha biblio-level item type. Set 'DEFAULT' to set a single value
```

Comment:
Maybe this could say "Set 'DEFAULT' to set a fallback value that will be used for any item types not given a more specific setting." Or something like that?

Reply:
thanks @demiankatz ! Appreciate all your help.

I've added this wording change in my latest follow-up commit.